### PR TITLE
currency service

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- currency service

## Trello Card
N/A

## Screenshot
Screen Shot 2019-12-10 at 10 29 30 AM

### CODE REVIEW

File: plink-frontend/src/environments/environment.ts [hunk 1]
```diff
@@ -3,7 +3,10 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  ROOT_URL: 'https://bravenewcoin-v1.p.rapidapi.com',
+  RAPID_API_HOST: 'bravenewcoin-v1.p.rapidapi.com',
+  RAPID_API_KEY: '9f338e8f57msh3c2d31501a8cd23p1b9fe6jsnd52601e97653',
 };
```
Comment (betoobernal) [inline plink-frontend/src/environments/environment.ts]: add environment to the .gitignore file

File: plink-frontend/src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'Plink Frontend';
+  pricess: any;
```
Comment (betoobernal) [inline plink-frontend/src/app/app.component.ts]: why is it `any` type?

File: plink-frontend/src/app/app.component.ts [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+  }
+
+  ngOnInit() {
+    this.getPrices();
+  }
+
+  getPrices() {
+    this.currencyService.getPrice().subscribe((data: any) => {
+      console.log(data)
+      this.pricess = data.prices;
```
Comment (betoobernal) [inline plink-frontend/src/app/app.component.ts]: you can use destructuring. Check this [article](https://codeburst.io/es6-destructuring-the-complete-guide-7f842d08b98f)

File: plink-frontend/src/app/app.component.ts (hunk 2 above)
Comment (jonnyalexbh) [reply plink-frontend/src/app/app.component.ts]: ready 🥇

File: plink-frontend/src/app/app.component.ts (hunk 3 above)
Comment (jonnyalexbh) [reply plink-frontend/src/app/app.component.ts]: ready 💯

File: plink-frontend/src/environments/environment.ts (hunk 1 above)
Comment (betoobernal) [reply plink-frontend/src/environments/environment.ts]: there exits some called [`glob matching`](https://github.com/begin/globbing) You could make this `/environments/**/*.ts`

**Review by betoobernal — APPROVED** [review]

**Pull request merged**